modules/virt-configuring-secondary-network-vm-live-migration.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * virt/live_migration/virt-migrating-vm-on-secondary-network.adoc
+
+[id="virt-configuring-secondary-network-vm-live-migration_{context}"]
+= Configuring a dedicated secondary network for virtual machine live migration
+
+To configure a dedicated secondary network for live migration, you must first create a bridge network attachment definition for a namespace in the CLI. Then add the name of the `NetworkAttachmentDefinition` object to the `HyperConverged` custom resource (CR).
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* The Multus Container Network Interface (CNI) plug-in is installed on the cluster.
+* Every node on the cluster has at least two Network Interface Cards (NICs), and the NICs to be used for live migration are connected to the same switch.
+* The virtual machine (VM) is running with eviction strategy `LiveMigrate`.
+
+.Procedure
+
+. Create a Linux bridge network attachment definition manifest.
```

Comment:
It's not a linux bridge
Just leave: Create `NetworkAttachmentDefinition` manifest